egs/swbd/s5c/local/chain/tuning/run_tdnn_7d.sh

```diff
@@ -133,7 +134,7 @@ if [ $stage -le 12 ]; then
     --ivector-dir exp/nnet3/ivectors_${train_set} \
```

Comment:
Please  bump up the number 7d->7e, and move the soft link.  That way we have an easy-to-see record of what you changed.  And then do (or somehow simulate the effects of doing)
`local/chain/compare_wer.sh  7d 7e`
so that we can see what you changed and what the WER changes were, from those files in tuning/.